homeassistant/helpers/entity.py

```diff
@@ -208,6 +226,7 @@ class EntityDescription:
     key: str
 
     device_class: str | None = None
+    # Type Literal (ENTITY_CATEG*) is deprecated as of 2021.12, use EntityCategory
```

Comment:
I'd remove the mention `(ENTITY_CATEG*)` from all the comments in the MR because the constants are not visible in any of them, and it's not only the constants that are deprecated but the general passing it as a `str`.

Reply:
Agreed, done.